packages/experiments/src/components/Stack/examples/Stack.Vertical.Spacing.Example.tsx

```diff
@@ -19,73 +24,73 @@ export class VerticalStackSpacingExample extends React.Component<{}, {}> {
         background: DefaultPalette.themePrimary,
         color: DefaultPalette.white
       }
-    });
+    };
 
     return (
       <Stack gap={10}>
         <Stack horizontal gap={50}>
           <Stack>
             <span>Numerical spacing</span>
-            <Stack className={styles.root} gap={10} padding={10}>
-              <span className={styles.item}>1</span>
-              <span className={styles.item}>2</span>
-              <span className={styles.item}>3</span>
+            <Stack styles={rootStyles} gap={10} padding={10}>
+              <span className={mergeStyleSets(itemStyles).root}>1</span>
```

Comment:
this feels really gross...

You do know about `mergeStyles` right? You don't need to use `mergeStyleSets` super redundantly like this!